tests/sentry/monitors/test_tasks.py

```diff
@@ -306,6 +306,7 @@ def test_missed_exception_handling(self, mark_environment_missing_mock):
         failing_monitor_environment = MonitorEnvironment.objects.create(
             monitor=exception_monitor,
             environment=self.environment,
+            last_checkin=ts - timedelta(minutes=2),
```

Comment:
When creating missed check ins for test we MUST have a `last_checkin`. Not having one is not reflective of the real world scenario